### PR TITLE
Cross-build 2.11 (Closes #65)

### LINKED ISSUE #65: 2.11.x support
Hi,

Thanks for the great lib. Would it be possible to have a 2.11.x version of this library published?

From what I can tell, it should just be a matter of adding 2.11.11 to the crossScalaVersions settings key, since Play 2.6.x still supports 2.11.

Thanks,

Lloyd

### PR DESCRIPTION
**Pull request by lloydmeta**
Closes #65 

* Add 2.11.x cross build
* Upgrade SBT version
* Bump Play 2.6 version to 2.6.6

### CODE REVIEW

**Comment by mosheeshel** [comment]
@AvihayTsayeg

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
+  - 2.12.3
+  - 2.11.11
 jdk:
   - oraclejdk8
 script:
-  - sbt clean coverage test coverageReport
+  - sbt ++${TRAVIS_SCALA_VERSION} clean coverage test coverageReport
 after_success:
   - bash <(curl -s https://codecov.io/bash)
+cache:
```
Comment (AvihayTsayeg) [inline .travis.yml]: We rarely run builds, the cache is currently unnecessary, can you remove this section

**Comment by AvihayTsayeg** [comment]
Squash the commit please.

File: .travis.yml (hunk 1 above)
Comment (lloydmeta) [reply .travis.yml]: I see it as tragedy of the commons + death by a thousand papercuts on behalf of Travis' and Maven Central's bandwidth, but suit yourself.

**Comment by lloydmeta** [comment]
Squashed.

**Comment by AvihayTsayeg** [comment]
@lloydmeta 
we will publish the new release  at he beginning of next week

**Comment by lloydmeta** [comment]
@AvihayTsayeg Thanks. Looking forward to it.

**Pull request merged**